### DISCUSSION TITLE
Total contributions from API

### DISCUSSION THREAD

**Discussion by arnelamo** [community/community#24581] [Apps, API and Webhooks]
<p>Hi. </p>
<p>I would like to pull my total contributions from GitHub, and display them on my personal site. However, I can’t find the right path in the v4 API for this information. I would basically like to get the same data as is shown in the “contribution calendar” on my GitHub profile.</p>
<p>Could someone push me in the right direction on this one?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/arnelamo">@arnelamo</a>, this is what you are looking for: <a href="https://developer.github.com/v4/object/contributioncalendar/" rel="nofollow noopener">https://developer.github.com/v4/object/contributioncalendar/</a></p>

**Comment by arnelamo** [comment]
<p>Thanks <a class="mention" href="/logankilpatrick">@logankilpatrick</a> !</p>

**Comment by ytrewqfh1** [comment]
<p>me when someone replies</p>
